test/e2e/util.go

```diff
@@ -1102,6 +1122,20 @@ func RunRC(config RCConfig) error {
 
 		Logf("%v %v Pods: %d out of %d created, %d running, %d pending, %d waiting, %d inactive, %d unknown ",
 			time.Now(), rc.Name, len(pods), config.Replicas, running, pending, waiting, inactive, unknown)
+
+		runningMetric.Set(float64(running))
```

Comment:
@timothysc okay to solve this later when we abstract metrics further?  For now gaurding in the if statement probably won't buy us anything...